easybuild/easyconfigs/i/Iris/Iris-1.11.0-intel-2016a-Python-3.5.1.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'PythonPackage'
+
+name = 'Iris'
+version = '1.11.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://scitools.org.uk/iris/'
+description = """Iris seeks to provide a powerful, easy to use, and community-driven Python library for analysing and visualising meteorological and oceanographic data sets."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+source_urls = ['https://github.com/SciTools/Iris/archive/']
+sources = ['v%(version)s.tar.gz']
+
+dependencies = [
+    ('Python', '3.5.1'),
+    ('Cartopy', '0.14.2', '-Python-%(pyver)s'),
+    ('pyke', '1.1.1', '-Python-%(pyver)s'),
+    ('Biggus', '0.13.0', '-Python-%(pyver)s'),
+    ('cf_units', '1.1.3', '-Python-%(pyver)s'),
+    ('netCDF', '4.3.3.1'),
+    ('grib_api', '1.16.0'),
+    # grib_api python bindings are needed for iris_grib extension
+    # ('grib_api', '1.16.0', '-Python-%(pyver)s'),
+]
+
+exts_defaultclass = 'PythonPackage'
+exts_filter = ("python -c 'import %(ext_name)s'", '')
+
+exts_list = [
+    # iris_grib doesn't work because grib_api python bindings don't work
```

Comment:
current grib_api easyconfig doesn't build python-bindings. So I added my own - which I didn't include in this PR since I haven't figured out the problem yet.  But I can add it.  Problem is that grib_api Python bindings to build, but cannot be imported.  I'll provide more info later.